README.md

```
@@ -50,6 +52,10 @@ Company | Duration | Team / Position | Form
 
 Company        | Project                | Skills              | Special Note
 ---------------|------------------------|---------------------|---------------------
+버블탭 | 리브랜딩 — 디자인 시스템(TDS) + SwiftUI 점진 전환 | Swift, Design System, Design Token, SwiftUI, UIKit | Rebranding, Design System, SwiftUI, UIKit/SwiftUI Parity
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: trailing_only; Actual: no_leading_or_trailing; Missing trailing pipe

(MD055, table-pipe-style)